Nero/src/NeroMet.cpp

```diff
@@ -75,10 +75,11 @@ int NeroMet::analyze(const edm::Event& iEvent){
         *metNoHF = TLorentzVector( metnohf.px(),metnohf.py(),metnohf.pz(),metnohf.energy());
         sumEtRawNoHF = metnohf.uncorSumEt();
 
-        const pat::MET &puppi = handle_puppi->front(); 
+        auto &puppi = handle_puppi->front(); 
         *metPuppi =  TLorentzVector( puppi.px(), puppi.py(),puppi.pz(),puppi.energy() );
-        sumEtRawPuppi = puppi.uncorSumEt();
+        sumEtRawPuppi = puppi.sumEt();
```

Comment:
Are you sure that this is uncorrected for JEC ? if it is not change, and you need it, change the name to SumEtPuppi